tests/e2e/backup_restore_suite_test.go

```diff
@@ -112,11 +114,13 @@ var _ = Describe("AWS backup restore tests", func() {
 			err = createRestoreFromBackup(vel.Client, namespace, backupName, restoreName)
 			Expect(err).ToNot(HaveOccurred())
 			Eventually(isRestoreDone(vel.Client, namespace, restoreName), timeoutMultiplier*time.Minute*4, time.Second*10).Should(BeTrue())
-			Expect(getVeleroContainerFailureLogs(vel.Namespace)).To(Equal([]string{}))
 
 			// Check if restore succeeded
 			succeeded, err = isRestoreCompletedSuccessfully(vel.Client, namespace, restoreName)
```

Comment:
This would never return `false, nil`.
It would always print an error here but the error will not contain pod logs.

pod logs should be collected and printed out either in these functions when returning false, or before running these functions.